docs/user-guide/rug/generators.md

```diff
@@ -129,10 +202,59 @@ highly recommended.
 We define the class that will implement our generator on line 8.  A
 generator implements the `#!typescript PopulateProject` interface.
 This interface requires the `#!typescript populate(Project)` method to
-be defined, which we do on line 20 (more on that below).  It is
+be defined, which we do on line 10 (more on that below).  It is
 convention for the generator and the class that implements it to have
 the same name.
 
+In the last line of the generator script we export an instance of that
+generator to make it visible to the Rug runtime when it executed (line 15).
+Like the generator class name, the name of the `#!typescript const`
+does not matter, but it is convention to use the generator/class name,
+lower-casing the first letter.
+
+At this stage, the generator does not do anything useful. As explained earlier,
```

Comment:
Not entirely true, you do end up with an exact copy of the original project with just these contents.